Source/Atomic/UI/UIMenuWindow.h

```diff
@@ -50,6 +51,8 @@ class UIMenuWindow : public UIWidget
 private:
 
     tb::TBSelectItemSource* source_;
+    
+    void FixMenu( tb::TBMenuWindow* ); // get rid of scrollbars for menu popups
```

Comment:
I'll remove the function, at the beginning I thought it was going to be reusable, but it turned into something else.